### ISSUE TITLE
[a11y] Cannot view CO form 

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1638]
![image](https://user-images.githubusercontent.com/37256050/168884175-e8750f46-c64f-4fc0-8ffb-3df340f0ecaf.png)

I am getting the error `Error: can't setup picklist for unknown field CollectingEventAttribute.GPSDatum`

db: `rbge_herb`
user: `robyn`

**Comment by maxxxxxdlp** [comment]
`edge` has the same error, except it is silently shown in the console rather than being so visible
![Screen Shot 2022-05-17 at 13 55 27](https://user-images.githubusercontent.com/40512816/168889180-e6323c78-c543-4323-817b-f1d68cfcebfc.png)

Silencing the error is not good, but has to be done in this case because specify 6 may have special handling for that field

**Comment by grantfitzsimmons** [comment]
I think this is in line with https://github.com/specify/specify7/issues/1616 - this could be an error that appears and the user can dismiss it. 

Maybe next release this can be implemented with a "Do not show again" checkbox or something
